File: src/main/java/org/customer/controller/CustomerViewController.java
package org.customer.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/customers")
public class CustomerViewController {

    @GetMapping("/details")
    public String customerDetailsPage() {
        return "customers"; // Refers to customers.html
    }
}

File: src/main/java.kotlin/org/customer/controller/CustomerViewController.kt
package org.customer.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/customers")
class CustomerViewController {
    @GetMapping("/details")
    fun customerDetailsPage(): String {
        return "customers" // Refers to customers.html
    }
}

